plugins/generator-1.21.1/neoforge-1.21.1/procedures/world_send_chat_advanced.java.ftl

```diff
@@ -0,0 +1,6 @@
+if (world instanceof ServerLevel _level)

```

Comment:
This is a bit cluttered code, and also when adding procedure block, it is not supposed to be a mask for a command content. If you want to make a procedure block to replace command, it should directly call underlying systems that a command invocation calls, to avoid the overhead of command parsing and while command system the current approach does.